framework/src/media/MediaWorker.cpp

```diff
@@ -22,7 +22,7 @@
 namespace media {
 	
 
-MediaWorker::MediaWorker() : mIsRunning(false), mRefCnt(0)
+MediaWorker::MediaWorker() : mStacksize(PTHREAD_STACK_DEFAULT), mThreadName("MediaWorker"), mIsRunning(false), mRefCnt(0)
```

Comment:
Member initialization methods differ in mideawork and playerwork
Is there a reason?

Reply:
mStacksize, mThreadName are member of MediaWorker. So we can't initialize like below
```
PlayerWorker::PlayerWorker() : mStacksize(4096)
```